doc/sphinx/changes.rst

```diff
@@ -853,6 +853,64 @@ Changes in 8.12+beta1
   (`#9803 <https://github.com/coq/coq/pull/9803>`_,
   by Laurent Théry and Michael Soegtrop).
 
+**Reference manual**
+
+- **Changed:**
+  The reference manual has been restructured.  In the new version,
+  there are less top-level chapters, and, in the HTML format, chapters
+  are split on multiple pages.  This is still a work-in-progress and
+  further restructuring is expected in the next versions of Coq
```

Comment:
I'd include a little bit of why as well as what was done.

Mention that sections were reordered to give a more logical organization.

For HTML, split on multiple pages to improve usability.  Or say "split into smaller pages".

Wording corrections:
```suggestion
  The reference manual has been restructured.  In the new version,
  there are fewer top-level chapters, and, in the HTML format, chapters
  are split into multiple pages.  This is still a work in progress and
  further restructuring is expected in the next versions of Coq
```

Reply:
Motivations were manyfolds (as expressed in the CEP, which this changelog entry links to) so it seems hard to summarize it shortly, but I'll try.